app/Models/SizeVariant.php

```diff
@@ -118,6 +108,7 @@ class SizeVariant extends Model
 		'photo_id', // see above
 		'short_path',  // serialize url instead
 		'sym_links', // don't serialize relation of symlinks
+		'',
```

Comment:
todo: nuke this.

Reply:
[fc7a773](https://github.com/LycheeOrg/Lychee/pull/2381/commits/fc7a773d63ca0d37bb8a77f8f4c08bd49ef31023)